apps/admin-ui/src/component/reservations/ReservationsDataLoader.tsx

```diff
@@ -16,20 +16,24 @@ import { fromUIDate, toApiDate } from "common/src/common/util";
 import { filterNonNullable, toNumber } from "common/src/helpers";
 import { useSearchParams } from "react-router-dom";
 
-function transformPaymentStatusSafe(t: string): OrderStatus | null {
+function transformPaymentStatusSafe(
+  t: string
+): OrderStatusWithFree | "free" | null {
```

Comment:
`"free"` is unnecessary

Reply:
That's weird, I'm sure I deleted "free" as a return type from that already, and my local branch doesn't show it.. 😮

Reply:
..fixed anyhow now - not that I actually changed anything 😅